src/main/java/br/com/sbk/sbking/core/Score.java

```diff
@@ -39,6 +39,22 @@ private void addEastWest(Trick trick) {
         eastWestPoints += this.scoreable.getPoints(trick);
     }
 
+    public void subtractTrickFromDirection(Trick trick, Direction winner) {
+        if (winner.isNorthSouth()) {
+            subtractNorthSouth(trick);
```

Comment:
I think those private methods are small enough (and not reused enough) to be inlined. But that is your call.

Reply:
I accepted your suggestion.